### ISSUE TITLE
MaxListenersExceededWarning

### ISSUE THREAD

**Issue by dotsinspace** [cody-greene/node-rabbitmq-client#48]
```
9|index  | (Use `node --trace-warnings ...` to show where the warning was created)
9|index  | MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 drain listeners added to [Socket]. Use emitter.setMaxListeners() to increase limit
9|index  |     at _addListener (node:events:601:17)
9|index  |     at Socket.addListener (node:events:619:10)
9|index  |     at Readable.on (node:internal/streams/readable:887:35)
9|index  |     at new EncoderStream (/home/smsflash/smsflash__node/.build/node_modules/.pnpm/rabbitmq-client@4.5.3/node_modules/rabbitmq-client/lib/util.js:113:13)
9|index  |     at new Channel (/home/smsflash/smsflash__node/.build/node_modules/.pnpm/rabbitmq-client@4.5.3/node_modules/rabbitmq-client/lib/Channel.js:90:21)
9|index  |     at Connection.acquire (/home/smsflash/smsflash__node/.build/node_modules/.pnpm/rabbitmq-client@4.5.3/node_modules/rabbitmq-client/lib/Connection.js:110:20)
9|index  |     at Consumer._setup (/home/smsflash/smsflash__node/.build/node_modules/.pnpm/rabbitmq-client@4.5.3/node_modules/rabbitmq-client/lib/Consumer.js:223:46) {
9|index  |   emitter: null,
9|index  |   type: 'drain',
9|index  |   count: 11
9|index  | } { hide_meta: true }

```

why i am getting this error ?. No idea

**Comment by cody-greene** [comment]
It's a warning that you can safely ignore, in this case. I'll publish a fix soon.

**Comment by cody-greene** [comment]
Fixed in v4.5.4

**Comment by dotsinspace** [comment]
Okay
